evap/staff/tests/test_forms.py

```diff
@@ -142,14 +142,14 @@ def test_results_cache_refreshed(self):
 
         results_after = collect_results(evaluation)
 
-        self.assertEqual(
-            results_before.contribution_results[0].contributor.first_name,
-            "Peter"
+        self.assertCountEqual(
+            (result.contributor.first_name for result in results_before.contribution_results if result.contributor),
+            ("Peter", ),
         )
 
-        self.assertEqual(
-            results_after.contribution_results[0].contributor.first_name,
-            "Patrick"
+        self.assertCountEqual(
+            (result.contributor.first_name for result in results_after.contribution_results if result.contributor),
+            ("Patrick", ),
```

Comment:
are you sure you want assertCountEqual? the actual string content is not compared anymore.

Reply:
`assertCountEqual` is quite a misleading name - see https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual. It does not assert that the lengths are the same, but rather that each element has the same count in the sequence

Reply:
right. thanks!